### PR TITLE
Update kfctl-k8s-istio.md

### PR DESCRIPTION
**Pull request by swiftdiaries**
Update script for quickly curling the latest release.
Similar to kustomize: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md#quickly-curl-the-latest

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1044)

### CODE REVIEW

File: content/docs/started/k8s/kfctl-k8s-istio.md [hunk 1]
```diff
@@ -12,10 +12,17 @@ This config creates a vanilla deployment of Kubeflow with all its core component
 
 Follow these steps to deploy Kubeflow:
 
-1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
+1. Download the latest `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
 
     ```
```
Comment (sarahmaddox) [inline content/docs/started/k8s/kfctl-k8s-istio.md]: Thanks for this update! I'd like to give people the option of doing the download and unpack manually if they prefer.

Suggestion: Leave the original instructions in place but offer your option to do the download and unpack in just 2 commands.

**Comment by sarahmaddox** [comment]
Friendly ping @swiftdiaries please would you take a look at my review comment.

File: content/docs/started/k8s/kfctl-k8s-istio.md (hunk 1 above)
Comment (swiftdiaries) [reply content/docs/started/k8s/kfctl-k8s-istio.md]: Thanks for the review Sarah! 
Sorry for the late reply

Comment (swiftdiaries) [reply content/docs/started/k8s/kfctl-k8s-istio.md]: Does this look good?

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1044--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-k8s-istio/

File: content/docs/started/k8s/kfctl-k8s-istio.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/started/k8s/kfctl-k8s-istio.md]: Thanks!

**Pull request merged**